recipes/mpc/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MpcConan(ConanFile):
+    name = "mpc"
+    description = "GNU MPC is a C library for the arithmetic of complex numbers with arbitrarily high precision " \
+                  "and correct rounding of the result"
+    topics = ("conan", "mpc", "multiprecision", "math", "mathematics")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.multiprecision.org/mpc/home.html"
+    license = "LGPL-3.0-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    requires = "mpfr/4.0.2"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.compiler == 'Visual Studio':
```

Comment:
```suggestion
        if self.settings.compiler == "Visual Studio":
```

Reply:
is there any advantage?

Reply:
No, this was the only usage of single quotes. And it was sticking in my eyes :smile: